### PR TITLE
Adding isMinimalAge boolean as parameter to HandleDate function

### PR DESCRIPTION
**Pull request by drcabral**
I did a small refactor to this function as well, in addition to the parameter changes.

But i saw that you don't have unit tests folder in this project, so i suggest that you review carefully my changes, ok?

This PR solves the issue #13

### CODE REVIEW

File: base/src/main/java/br/com/redcode/base/extensions/ExtensionTextView.kt [hunk 1]
```diff
@@ -82,4 +76,23 @@ fun TextView.updateBadge(count: Int) {
...
+        calendar.set(Calendar.YEAR, year)
+        calendar.set(Calendar.MONTH, month)
+        calendar.set(Calendar.DAY_OF_MONTH, dayOfMonth)
+
+        val result = calendar.to__dd_MM_yyyy()
+
+        text = result
+        callback?.let { result }
+
+        setTextColor(ContextCompat.getColor(context, R.color.black))
```
Comment (pedrofsn) [inline base/src/main/java/br/com/redcode/base/extensions/ExtensionTextView.kt]: Change to 'android.R.color.black'

File: base/src/main/java/br/com/redcode/base/extensions/ExtensionTextView.kt [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package br.com.redcode.base.extensions
 
+import android.R
```
Comment (pedrofsn) [inline base/src/main/java/br/com/redcode/base/extensions/ExtensionTextView.kt]: Remove this import

**Review by pedrofsn — CHANGES_REQUESTED** [review]
Hi @ drcabral, we need some fixes to project keep running.

File: base/src/main/java/br/com/redcode/base/extensions/ExtensionTextView.kt (hunk 1 above)
Comment (drcabral) [reply base/src/main/java/br/com/redcode/base/extensions/ExtensionTextView.kt]: Done!

File: base/src/main/java/br/com/redcode/base/extensions/ExtensionTextView.kt (hunk 2 above)
Comment (drcabral) [reply base/src/main/java/br/com/redcode/base/extensions/ExtensionTextView.kt]: Done!

**Comment by pedrofsn** [comment]
Thanks for contributing!

**Pull request merged**